docs/api/reference/eth-sendbundle.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: eth_sendBundle
+description: Send a bundle of signed transactions to be executed atomically.
+---
+
+# `eth_sendBundle`
+
+:::note
+
+Access to this endpoint is permissioned via Infura, and is not relevant for node operators as it's 
+available on sequencers only.
+
+:::
+
+Privately sends a bundle of signed transactions to be executed atomically, allowing the intended 
+transaction to be "sandwiched" between others, enabling services such as:
+- Frontrunning protection
+- Gas abstraction
+- Revert protection
+
+## Parameters
+
+- `txs`: An array of signed transactions to execute in the bundle.
```

Comment:
@jlwllmr, does this not also accept optional parameters such as `minTimestamp`, `maxTimestamp` etc?
See: https://github.com/Consensys/linea-sequencer/pull/142/files#diff-a006c66ef3ccf456a6e22bea684b1329d7a70bc04ebefea14bf0752698cdc64aR93-R103

Reply:
that link would suggest so! 😅  I only listed these two as I was under the impression those were the only ones we were using. @julien-marchand could you advise?